src/main/java/filter/expression/QualifierType.java

```diff
@@ -93,4 +93,32 @@ private static boolean isSpecialQualifier(String input) {
     public String toString() {
         return name().toLowerCase();
     }
+    
+    /**
+     * Retrieves acceptable inputs of a qualifier 
```

Comment:
Maybe this could be something like: "returns a short description of the class of valid inputs a type of qualifier may be used with".